zipkin-web/src/main/resources/app/js/main.js

```diff
@@ -1,4 +1,10 @@
-'use strict';
+import {compose, registry, advice, logger as withLogging, debug} from 'flight';
```

Comment:
Finally, ES6 destructuring (similar to unapply/pattern matching in Scala) and harmony modules!